cluster/userdata-master.yaml

```diff
@@ -853,6 +853,59 @@ write_files:
               securityContext:
                 privileged: true
 
+
+  - path: /srv/kubernetes/manifests/daemonsets/cluster-autoscaler.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: Deployment
+      metadata:
+        name: cluster-autoscaler
+        namespace: kube-system
+        labels:
+          app: cluster-autoscaler
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            app: cluster-autoscaler
+        template:
+          metadata:
+            labels:
+              app: cluster-autoscaler
+            annotations:
+              scheduler.alpha.kubernetes.io/critical-pod: ''
+          spec:
+            hostNetwork: true
```

Comment:
This is only needed to get node IAM roles right? In that case I think it's better to leverage kube2iam and use a custom role to avoid "bloating" the node role with "random" roles.